pkg/config/configuration.go

```diff
@@ -60,6 +60,9 @@ const (
 	DefaultNamespace    = "default"
 	ActionPolicyApp     = "app"
 	ActionPolicyGlobal  = "global"
+
+	defaultMaxWorkflowConcurrentInvocations = 1000
+	defaultMaxActivityConcurrentInvocations = 1000
```

Comment:
have we tested this? Or should we for now set it back to 100? and later increase it once we have enough tests.

Reply:
@cgillum is there a reason for the increase in concurrent invocations?

Reply:
I'm fine with reverting it back and increasing it later.